parachains/runtimes/assets/asset-hub-kusama/src/lib.rs

```diff
@@ -278,6 +285,75 @@ impl pallet_assets::Config<TrustBackedAssetsInstance> for Runtime {
 	type BenchmarkHelper = ();
 }
 
+parameter_types! {
+	pub const AssetConversionPalletId: PalletId = PalletId(*b"py/ascon");
+	pub storage AllowMultiAssetPools: bool = false;
```

Comment:
what's the reason of using `storage` here?

Reply:
Yeah, `const` is probably fine.